lib/gemfury/client.rb

```diff
@@ -41,6 +41,13 @@ def versions(name, options = {})
       checked_response_body(response)
     end
 
+    def package(name, options = {})
+      ensure_ready!(:authorization)
+      url = "packages/#{escape(name)}"
+      response = connection(:api_version => 2).get(url, options)
```

Comment:
API v2 is not stable and shouldn't be used in the CLI. This particular endpoint, for example, is deprecated and is no longer used by the Dashboard. Better to add the needed details to API v1.